File: boat-bay-frontend/src/app/models/boat-product.ts
import { BoatLintReport } from "./boat-lint-report";
import { BoatStatistics } from "./boat-dashboard";

export interface BoatProduct {
  portalKey: string;
  portalName: string;

  id: number;
  key: string;
  name: string;
  content: number;

  createdOn: Date;
  createdBy: String;

  lastLintReport: BoatLintReport;

  statistics: BoatStatistics;

  jiraProjectId: string;
}

File: boat-bay-server/src/main/java/com/backbase/oss/boat/bay/web/views/dashboard/models/BoatProduct.java
package com.backbase.oss.boat.bay.web.views.dashboard.models;

import com.backbase.oss.boat.bay.service.statistics.BoatStatistics;
import java.time.LocalDateTime;
import lombok.Data;

@Data
public class BoatProduct {


    private String portalKey;
    private String portalName;

    private String id;
    private String key;
    private String name;
    private String content;

    private LocalDateTime createdOn;
    private String createdBy;

    private BoatLintReport lastLintReport;

    private BoatStatistics statistics;

    private String jiraProjectId;

}

